### ISSUE TITLE
Add warning for pip2 users or possibly change documentation

### ISSUE THREAD

**Issue by DougBeney** [wagtail/wagtail#4327]
Congrats on version 2!

Since you are completely on Python3 now, I believe [the documentation](https://wagtail.io/developers/) should either instruct the user to use `pip3` or show a warning when `pip2` is used instead.

I'm not 100% sure if it's possible for you to show a warning for `pip2`, but I just thought I'd suggest it.

For some systems -- mostly Linux distributions -- the `pip` command defaults to using `pip2`.

It turns out my system, running on Solus Linux, defaults to pip2 instead of pip3. So, I unknowingly installed Wagtail using pip2. When I used the `wagtail start` command to create a site, it created this unobvious error:

```
Traceback (most recent call last):
  File "/usr/bin/wagtail", line 11, in <module>
    load_entry_point('wagtail==2.0', 'console_scripts', 'wagtail')()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 572, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2755, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2408, in load
    return self.resolve()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2414, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python2.7/site-packages/wagtail/bin/wagtail.py", line 230
    print(line, end='')  # NOQA
                   ^
SyntaxError: invalid syntax
```

I fixed this issue by uninstalling wagtail and using `pip3 install wagtail` instead.

**Comment by tomdyson** [comment]
Thanks @DougBeney! I'm not sure about the best way to address this. I don't have a `pip3` on my OS X terminal; I think the most reliable method is the slightly cumbersome

`python3 -m pip install wagtail`

but others may have better ideas.

**Comment by DougBeney** [comment]
What are your thoughts about this solution, implementing a warning?

First, you'd create a file called something like `versioncheck.py`:

```
import sys
if sys.version_info.major < 3:
    sys.stdout.write("Wagtail requires Python 3 and also to be installed using pip3.\n")
    sys.exit(1)
```

Then, you can simply import this `versioncheck.py` at the top of whichever file of yours is in charge of the wagtail command

**Comment by gasman** [comment]
Fixed in #4416
